### PR TITLE
rzpi: update for rc2

### PR DESCRIPTION
**Pull request by vudangRVC**
Update points:
- Point to https URL of meta-renesas layer and rc2 tag.
- Build Chromium with HW codec and GPU support.

Please check.

### CODE REVIEW

File: rz-sbc/rzsbc_yocto.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 # -----------------------------Yocto package------------------------------------
 # Make sure that the following packages have been downloaded from the official website
 # RZ/G Verified Linux Package [5.10-CIP] V3.0.5 update1
 REN_LINUX_BSP_PKG="RTK0EF0045Z0021AZJ-v3.0.5-update1"
 REN_LINUX_BSP_META="rzg_vlp_v3.0.5"
 REN_LOCAL_META="meta-renesas"
-REN_LOCAL_REPO="git@github.com:Renesas-SST/meta-renesas.git"
-REN_LOCAL_BRANCH="dunfell/rz-sbc"
+REN_LOCAL_REPO="https://github.com/Renesas-SST/meta-renesas.git"
+REN_LOCAL_BRANCH="rzg2l-sbc-qt-demo-rc2"
```
Comment (preetam-reddy) [inline rz-sbc/rzsbc_yocto.sh]: point this to dunfell/rz-sbc and change it later while building.

File: rz-sbc/rzsbc_yocto.sh [hunk 2]
```diff
@@ -126,15 +126,12 @@ function unpack_bsp(){
...
+
 	git clone https://github.com/OSSystems/meta-browser.git
 	cd meta-browser
-	git checkout e75da7cc83393b174220b808d96f8f05ab7ad4d2
-	cd ../
-	rm -rf meta-openembedded/
-	git clone https://github.com/openembedded/meta-openembedded
-	cd meta-openembedded
-	git checkout 116bfe8d5e5851e7fc5424f40da8691a19c5b5ee
+	git checkout dunfell
```
Comment (preetam-reddy) [inline rz-sbc/rzsbc_yocto.sh]: This has no meaning.

File: rz-sbc/rzsbc_yocto.sh (hunk 1 above)
Comment (vudangRVC) [reply rz-sbc/rzsbc_yocto.sh]: I didn't get it. Do I need to define one more variable for the build tag?

Like: 
REN_LOCAL_BRANCH="dunfell/rz-sbc"
REN_LOCAL_TAG="rzg2l-sbc-qt-demo-rc2"

then checkout the tag later?

File: rz-sbc/rzsbc_yocto.sh (hunk 2 above)
Comment (vudangRVC) [reply rz-sbc/rzsbc_yocto.sh]: I agree

File: rz-sbc/rzsbc_yocto.sh (hunk 1 above)
Comment (preetam-reddy) [reply rz-sbc/rzsbc_yocto.sh]: after merge, this should checkout dunfell/rz-sbc. 
rzg2l-sbc-qt-demo-rc2 will be deleted after the merge.

File: rz-sbc/rzsbc_yocto.sh (hunk 2 above)
Comment (preetam-reddy) [reply rz-sbc/rzsbc_yocto.sh]: Then remove it.

File: rz-sbc/rzsbc_yocto.sh (hunk 1 above)
Comment (vudangRVC) [reply rz-sbc/rzsbc_yocto.sh]: updated

File: rz-sbc/rzsbc_yocto.sh (hunk 2 above)
Comment (vudangRVC) [reply rz-sbc/rzsbc_yocto.sh]: updated

**Pull request merged**